src/pages/Proposal/ProposalSummary/index.tsx

```diff
@@ -157,6 +161,26 @@ export const ProposalSummaryUi: React.FC<
     // no default
   }
 
+  const countdownData = useMemo(() => {
+    if (state === 'Active' && endDate) {
+      return {
+        date: endDate,
+        // DO NOT REMOVE COMMENT: needed by i18next to extract translation key
+        // t('voteProposalUi.activeUntilDate')
+        i18nKey: 'voteProposalUi.activeUntilDate',
+      };
+    }
+
+    if (state === 'Succeeded' && proposalEta && isAfter(proposalEta, new Date())) {
+      return {
+        date: proposalEta,
+        // DO NOT REMOVE COMMENT: needed by i18next to extract translation key
+        // t('voteProposalUi.readyToQueueAt')
```

Comment:
Do you mean doing something like this?
<img width="876" alt="Screen Shot 2022-07-14 at 15 24 43" src="https://user-images.githubusercontent.com/44675210/179005650-cba07c69-9c80-422e-95f5-b26dcab9ea11.png">
Because in this case we'd be duplicating the component itself, when only the i18nkey changes

Reply:
I was thinking about extracting the props into variables at the top of the function to reduce the duplication, because I understand this was done to reduce duplication. Either way looks good to me, I was just thinking how we can avoid having comments that affect code

Reply:
I see. Yeah I'd rather keep the comment, because this way we clearly indicate only the translation key changes